simulator/examples/hello-world.rs

```diff
@@ -1,3 +1,7 @@
+//! # Example: Hello world
```

Comment:
I've just noticed an error in a comment below. I can't add a comment on that line because it's not part of your changes, but could you fix it in this PR?
```
// Draw a square with a 3px wide stroke.
```
should be

```
// Draw a circle with a 3px wide stroke.
```
This also applies to the copy in README.md.